tests/sentry/api/endpoints/test_scim_schema.py

```diff
@@ -0,0 +1,13 @@
+from django.urls import reverse
+
+from .test_scim import SCIMTestCase
+
+
+class SCIMSchemaEndpointTest(SCIMTestCase):
+    def test_schema_200s(self):
+        url = reverse(
+            "sentry-api-0-organization-scim-schema-index",
+            args=[self.organization.slug],
+        )
+        response = self.client.get(url)
+        assert response.status_code == 200, response.content
```

Comment:
just a basic 200 check since all we're doing is creating a schema and returning it -- checking the schema of the response seemed not too useful here.